docs/core/docker/building-net-docker-images.md

```diff
@@ -11,10 +11,13 @@ ms.technology: dotnet-docker
 ms.devlang: dotnet
 ms.assetid: 03c28597-7e73-46d6-a9c3-f9cb55642739
 ---
- 
 
 #Building Docker Images for .NET Core Applications
 
+ 
+> [!IMPORTANT]
+> We are in the process of updating for .NET Core 2.0. The instructions below are out of date. We sincerely apologize for any inconvenience!
```

Comment:
nit: sorry for not catching this earlier. Can we say in the process of updating this topic/article?

we're also not supposed to use below. So replace it with "The following instructions"